src/platform/mbed/net/include/common.h

```diff
@@ -0,0 +1,26 @@
+#ifndef MBED_NET_COMMON_H
+#define MBED_NET_COMMON_H
+
+#include <errno.h>
+#include <stdlib.h>
+
+#ifdef __cplusplus
+extern "C" {
+#endif
+
+#ifndef _set_errno
+#define _set_errno(err)                                                                                                            \
```

Comment:
What's the point of having `_set_errno` instead of assigning `errno` ?

Reply:
Just to have a common approach for setting errno. I should change the name to remove `_`